src/adapters/bidderFactory.js

```diff
@@ -176,8 +194,30 @@ export function newBidder(spec) {
       // After all the responses have come back, call done() and
       // register any required usersync pixels.
       const responses = [];
+
       function afterAllResponses(bids) {
         const bidsArray = bids ? (bids[0] ? bids : [bids]) : [];
+        // bidderRequest.end = timestamp();
+        let bidIds = bidsArray.map(bid => {
+          if (bid.requestId) {
+            return bid.requestId.toString();
```

Comment:
Why are you converting bid ids to strings?  Seems inefficient and unecessary

Reply:
Existing unit tests do not consistently use strings or numeric values for id's. I was unsure if this was an issue with the tests or in place to make sure ids can be provided either as a string or int.  In any case this is safer.

I could write this using a double equal, but then the cast would be implicit instead of explicit.